.gitignore

```diff
@@ -28,3 +28,4 @@ _build
 # Dependency directory
 # https://www.npmjs.org/doc/misc/npm-faq.html#should-i-check-my-node_modules-folder-into-git
 node_modules
+yarn.lock
```

Comment:
Was this meant to be committed?

Reply:
Yes. I should have used a different commit though.